package.json

```diff
@@ -31,7 +31,7 @@
     "@ngrx/effects": "^9.2.0",
     "@ngx-translate/core": "^11.0.1",
     "@ngx-translate/http-loader": "^4.0.0",
-    "@stomp/rx-stomp": "^1.0.1",
+    "@stomp/ng2-stompjs": "^8.0.0",
```

Comment:
I do not fully understand why these changes will take place?

Reply:
the test failed. maybe this version that works properly with LinuxOS. After replacement, the test did not failed